internal/services/gateway/gateway.go

```diff
@@ -292,6 +298,7 @@ func (g *Gateway) Run(ctx context.Context) error {
 	apirouter.Handle("/users/{userref}", authForcedHandler(deleteUserHandler)).Methods("DELETE")
 	apirouter.Handle("/user/createrun", authForcedHandler(userCreateRunHandler)).Methods("POST")
 	apirouter.Handle("/user/orgs", authForcedHandler(userOrgsHandler)).Methods("GET")
+	apirouter.Handle("/user/invitations", authForcedHandler(userOrgInvitationsHandler)).Methods("GET")
```

Comment:
It should represents that they're org invitations with something like `/user/org_invitations`